src/Tasks.UnitTests/AssemblyDependency/Miscellaneous.cs

```diff
@@ -3269,41 +3269,6 @@ public void ParentAssemblyResolvedFromAForGac()
             Assert.Equal(reference2.ResolvedSearchPath, parentReferenceFolders[0].Directory);
         }
 
-        /// <summary>
-        /// Generate a fake reference which has been resolved from the gac. We will use it to verify the creation of the exclusion list.
-        /// </summary>
-        /// <returns></returns>
-        private ReferenceTable GenerateTableWithAssemblyFromTheGlobalLocation(string location)
```

Comment:
it would be nicer to split the changes of the feature and code improvements, but fine